src/components/cart.js

```diff
@@ -428,7 +428,9 @@ export default class Cart extends Component {
         this.updateCache(this.model.lineItems);
         this.view.render();
         this.toggles.forEach((toggle) => toggle.view.render());
-        this.view.setFocus();
+        if (!openCart) {
```

Comment:
If `openCart` is true, it triggers a call to `cart.open()` above, which calls `setFocus` as well